test/Microsoft.IdentityModel.Validators.Tests/MicrosoftIdentityIssuerValidatorTest.cs

```diff
@@ -921,12 +1080,21 @@ public void Validate_UsesLKGWithConfigurationProvider()
 
             actualIssuer = v1AadIssuerValidator.Validate(v1TokenIssuer, v1JwtSecurityToken, new TokenValidationParameters { ValidateWithLKG = true });
             IdentityComparer.AreEqual(v1TokenIssuer, actualIssuer, context);
+
+            v1ConfigurationManager.RefreshedConfiguration = v1ConfigurationRefreshed;
+            v1ConfigurationManager.RequestRefresh();
+
+            actualIssuer = await v1AadIssuerValidator.ValidateAsync(v1TokenIssuer, v1JwtSecurityToken, new TokenValidationParameters { ValidateWithLKG = true });
+            IdentityComparer.AreEqual(v1TokenIssuer, actualIssuer, context);
             TestUtilities.AssertFailIfErrors(context);
 
             // validating cross versions also validates with LKG
             actualIssuer = aadIssuerValidator.Validate(v1TokenIssuer, v1JwtSecurityToken, new TokenValidationParameters { ValidateWithLKG = true });
+            IdentityComparer.AreEqual(v1TokenIssuer, actualIssuer, context);
 
+            actualIssuer = await aadIssuerValidator.ValidateAsync(v1TokenIssuer, v1JwtSecurityToken, new TokenValidationParameters { ValidateWithLKG = true });
             IdentityComparer.AreEqual(v1TokenIssuer, actualIssuer, context);
+
             TestUtilities.AssertFailIfErrors(context);
 
             // if LKG not valid validation fails
```

Comment:
There seem to be a few missing places in these files that didnt switch to async ? Why is that?

Reply:
^_^ good catch, I missed updating this block.